images/macos/toolsets/toolset-10.15.json

```diff
@@ -20,7 +20,7 @@
     "xamarin": {
         "vsmac": "latest",
         "mono-versions": [
-            "6.12.0.113", "6.10.0.106", "6.8.0.123", "6.6.0.166", "6.4.0.208"
```

Comment:
no need to update patch version of mono, we add new minor versions only

Reply:
done, thank you!